packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1,15 +1,14 @@
-use fuel_tx::{AssetId, ContractId, Receipt, Salt};
-use fuels::prelude::*;
+use fuel_tx::{AssetId, ContractId, Receipt};
+use fuels::prelude::Error;
+use fuels::prelude::{
+    setup_address_and_coins, setup_test_provider, setup_test_provider_and_wallet, CallParameters,
+    Contract, LocalWallet, Provider, Signer, TxParameters, DEFAULT_INITIAL_BALANCE,
+};
 use fuels_abigen_macro::abigen;
-use rand::rngs::StdRng;
-use rand::{Rng, SeedableRng};
+use fuels_core::constants::NATIVE_ASSET_ID;
+use fuels_core::Token;
 use sha2::{Digest, Sha256};
 
-/// Note: all the tests and examples below require pre-compiled Sway projects.
-/// To compile these projects, run `cargo run --bin build-test-projects`.
-/// It will build all test projects, creating their respective binaries,
-/// ABI files, and lock files. These are not to be committed to the repository.
```

Comment:
Why are we deleting this? This is still true and this comment is helpful to newcomers who don't know that they have to run `cargo run --bin build-test-projects`.

Reply:
I brought back these comments and now they are on the branch.